### ISSUE TITLE
Creating XG from GFA takes a long time for graphs whose IDs do not start at 1

### ISSUE THREAD

**Issue by jeizenga** [vgteam/xg#2]
I think the culprit is this line:
https://github.com/vgteam/xg/blob/master/src/xg.cpp#L591

**Comment by ekg** [comment]
Yes, that's why. This call will pad the id space. It needs to start at a
low number and be mostly contiguous for this to work comfortably.

There are many ways to fix this. A translation into the index space is one
mechanism that would be quick to implement.
